### PR TITLE
Detail

### PR DESCRIPTION
**Pull request by phamtin**

### CODE REVIEW

**Comment by phamtin** [comment]
ready

File: asset/css/style.css [hunk 1]
```diff
@@ -20,13 +20,18 @@ html {
...
-  font-weight: 400; }
+  color: #1d1d1d; }
   .para--pay {
     color: whitesmoke; }
+<<<<<<< HEAD
+=======
+  .para--detail {
+    font-size: 14px;
+    color: #505050; }
+>>>>>>> 0b80c71... detail page
```
Comment (chien-pv) [inline asset/css/style.css]: (tat8)

File: asset/css/style.css [hunk 2]
```diff
@@ -80,6 +88,11 @@ body {
     background-color: #6eb320 !important; }
   .custom-btn--update {
     background-color: #6eb320 !important; }
+=======
+.custom-btn--detail {
+  border-radius: 15px !important;
+  border: 2px #c9c9c9 solid !important; }
+>>>>>>> 0b80c71... detail page
```
Comment (chien-pv) [inline asset/css/style.css]: (tat8)

File: sass/components/_button.scss [hunk 3]
```diff
@@ -8,5 +9,14 @@
...
         background-color: $green-primary !important;
+=======
+.custom-btn{
+    // border-radius: 0!important;
+
+    &--detail{
+        border-radius: 15px!important;
+        border: 2px $color-grey solid!important;
+
+>>>>>>> 0b80c71... detail page
```
Comment (chien-pv) [inline sass/components/_button.scss]: fix confict vậy à, tat

**Comment by chien-pv** [comment]
commented

**Comment by phamtin** [comment]
ready

**Pull request merged**